### DISCUSSION TITLE
Auto3Dseg support for .mha files

### DISCUSSION THREAD

**Discussion by bilalcodehub** [Project-MONAI/tutorials#1710] [Q&A] [answered]
Dear MONAI Team,

I am interested in using Auto3Dseg for processing scans and labels stored in `.mha` format, but noticed that the example tutorials predominantly utilise the `nii.gz` file format. Could you please confirm if Auto3Dseg is compatible with `.mha` files as well? Additionally, if `.mha` files are not natively supported, could you suggest an optimal workaround to accommodate this format?

Thank you for your assistance.

Best regards,
Bilal

**Comment by KumoLiu** [comment, marked as answer]
Hi @bilalcodehub,

I think it can be opened as well since we have `ITKReader` which can read that format.
https://github.com/Project-MONAI/MONAI/blob/4af23069ccbbd26d7816bafc3762365ce4c5da77/monai/data/image_reader.py#L152

Thanks,
Yun

**Reply by bilalcodehub** [reply]
Dear Yun, Thank you for confirming that Auto3DSeg supports .mha files. 

Could you please clarify whether the library automatically detects and processes these files based on their extensions, or will we need to manually program this functionality?

Looking forward to your guidance.

Thanks,
Bilal

**Reply by KumoLiu** [reply]
`LoadImage` can automatically load date by your extension and existing package, you can try. Thanks.

**Reply by bilalcodehub** [reply]
Yes, the library works perfect with the `.mha` files as you described without making any changes to the code. Wonderful. Thanks.
